### PR TITLE
[UI > Settings] Fix You searching

### PR DESCRIPTION
**Pull request by nexpid**
also removed the youKey snakeCase mapping because it seems unnecessary (correct me if i'm wrong though)

### CODE REVIEW

**Comment by acquitelol** [comment]
I'm pretty sure they did it so that it was backwards compatible with panels UI, so please ensure panels UI pages still work

**Comment by nexpid** [comment]
@acquitelol just checked, still works

**Comment by acquitelol** [comment]
> @acquitelol just checked, still works

slayyy

**Review by redstonekasi — APPROVED** [review]

File: src/ui/settings/data.tsx [hunk 1]
```diff
@@ -127,10 +126,9 @@ export const getYouData = () => {
                 type: "route",
                 title: () => s.title,
                 icon: s.icon ? getAssetIDByName(s.icon) : null,
+                usePredicate: s.shouldRender,
```
Comment (amsryq) [inline src/ui/settings/data.tsx]: is there a chance we can also fix the developer settings not rendering here? so
`getLayout` would return all screens then `usePredicate: () => useProxy(settings) && s.shouldRender(),`

File: src/ui/settings/data.tsx [hunk 2]
```diff
@@ -106,14 +106,14 @@ export const getPanelsScreens = () => keyMap(getScreens(), (s) => ({
...
-    const screens = getScreens(true);
+    const screens = getScreens();
 
     return {
         getLayout: () => ({
             title: "Vendetta",
             label: "Vendetta",
             // We can't use our keyMap function here since `settings` is an array not an object
-            settings: getRenderableScreens(true).map(s => s.key)
+            settings: getRenderableScreens().map(s => s.key)
```
Comment (amsryq) [inline src/ui/settings/data.tsx:116]:
```suggestion
            settings: screens.map(s => s.key)
```

**Pull request closed without merging**